pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -876,10 +876,27 @@ void MainEmuFrame::Menu_Capture_Screenshot_Screenshot_As_Click(wxCommandEvent &e
 	if (!CoreThread.IsOpen())
 		return;
 
-	wxFileDialog fileDialog(this, "Select a file", g_Conf->Folders.Snapshots.ToAscii(), wxEmptyString, "PNG files (*.png)|*.png", wxFD_SAVE | wxFD_OVERWRITE_PROMPT);
+	// Ensure emulation is paused so that the correct image is captured
+#ifndef DISABLE_RECORDING
+	bool wasPausedViaInputRecTools = g_RecordingControls.IsEmulationAndRecordingPaused();
+	if (!wasPausedViaInputRecTools)
+		CoreThread.Pause();
+#else
+	CoreThread.Pause();
+#endif
```

Comment:
It would probably be best to to utilize only CoreThread.IsPaused() in this situation so we can limit input-recording related preprocessor conditions to only code directly related to recording.

Reply:
Ah that's true, I had that originally but went through a few iterations and forgot.